specs/release-automation/flow-for-automation.puml

```diff
@@ -0,0 +1,67 @@
+@startuml
+:User triggers the workflow;
+start
+:Create release files;
+while(unchecked release files)
+if (addHeader) then(true)
```

Comment:
Could you add the steps for actually creating the files? i.e. the CSV and Excel files?